test/lightning_web/end_to_end_test.exs

```diff
@@ -1,173 +1,278 @@
 # This module will be re-introduced in https://github.com/OpenFn/Lightning/issues/1143
 defmodule LightningWeb.EndToEndTest do
-  use LightningWeb.ConnCase, async: true
-  use Oban.Testing, repo: Lightning.Repo
+  use LightningWeb.ConnCase, async: false
 
   import Lightning.JobsFixtures
   import Lightning.Factories
 
+  alias Lightning.Attempt
+  alias Lightning.Attempts
+  alias Lightning.Attempts.Events
   alias Lightning.Invocation
+  alias Lightning.Repo
+  alias Lightning.WorkOrders
+  alias Lightning.Runtime.RuntimeManager
 
-  import Ecto.Query
+  require Attempt
```

Comment:
I think you meant `import Attempt`, but I'm guessing this wasn't supposed to be here.

Reply:
It's okay, it's for the macro to reuse the same list